src/import/ImportPlugin.cpp

```diff
@@ -69,5 +69,5 @@ sampleFormat ImportFileHandle::ChooseFormat(sampleFormat effectiveFormat)
 std::shared_ptr<WaveTrack> ImportFileHandle::NewWaveTrack(
    WaveTrackFactory &trackFactory, sampleFormat effectiveFormat, double rate)
 {
-   return trackFactory.NewWaveTrack(ChooseFormat(effectiveFormat), rate);
+   return trackFactory.Create(ChooseFormat(effectiveFormat), rate);
```

Comment:
Certain import file formats might contain multiple channels that become multiple tracks.  Should you consider giving them unique names?  But in the usual case of one track, I understand that you would not want to add the numeral.

Reply:
>Should you consider giving them unique names?

If I understand your question correctly, then I think `ProjectFileManager::AddImportedTracks` assigns unique names to the channels.

>But in the usual case of one track, I understand that you would not want to add the numeral.

Yes.